modules/messages/client/components/Thread.component.js

```diff
@@ -158,24 +159,63 @@ export default function Thread({ user, profileMinimumLength }) {
     }
   }
 
+  function createFakeUserObject(userId) {
+    return {
+      _id: userId,
+      displayName: 'Unknown User',
+      username: null,
+      member: [],
+      languages: [],
```

Comment:
I wonder if some of this can be avoided by doing checks like `user?.username`, `user?.languages?.length` in the code instead?